src/clipperFunctionalities.cpp

```diff
@@ -0,0 +1,32 @@
+#include <opencv2/core.hpp>
+#include <opencv2/imgproc.hpp>
+#include <opencv2/highgui.hpp>
+
+#include "clipper.hpp"
+
+using namespace ClipperLib;
+
+Paths enlarge(IntPoint* points, int offset){
+    Path subj;
+    Paths solution;
+    for(int i = 0; i < sizeof(points); i++){
+        subj->push_back(points[i]);
+    }
+    ClipperOffset co;
+    co.AddPath(subj, jtRound, etClosedPolygon);
+    co.Execute(solution, offset)
+    return solution;
+}
+
+void printSolution(Paths solution){
+    printf("solution size = %d\n", (int)solutions.size());
+    for(unsigned i = 0; i < solution.size(); i++){
```

Comment:
Does it work with just `unsigned`? I thought you had to write something like `unsigned int`
Maybe it's better if we add a simple test in the main file, just to make sure it actually works.

Reply:
It does, the code is taken directly from the professor. You can find some test on the examples put by the professor, I did not include them in the code as they were already tested by the professor.